images/metering-ansible-operator/roles/meteringconfig/library/k8s_event.py

```diff
@@ -199,9 +200,9 @@
        returned: success
        type: str
      source:
-       description: Component for reporting this Event
+       description: Component for reporting this Event. Includes source.
```

Comment:
nit: Let's mirror the description of [`corev1.EventSource`](https://github.com/kubernetes/api/blob/master/core/v1/types.go#L5057).

Reply:
`description: source contains information for an event.`